cs/commandstation/ProgrammingTrackFrontend.hxx

```diff
@@ -735,7 +737,10 @@ class ProgrammingTrackFrontend
     LOG(INFO, "CV railcom feedback ch=%d: %s", f.channel,
         railcom_debug(f).c_str());
     if (!f.ch2Size) {
-      return record_railcom_status(ERROR_NO_RAILCOM_CH2_DATA);
+      // Ignores this; leave in pending and maybe a later repeat reaches the
+      // decoder.
+      return;
+      //return record_railcom_status(ERROR_NO_RAILCOM_CH2_DATA);
```

Comment:
Is this dead code or are you keeping it around for some reason?

Reply:
fixed